src/utils.js

```diff
@@ -48,7 +48,6 @@ export const updateOnMutation = function (componentBeingObserved, callback) {
     observer.observe(componentBeingObserved, {
         attributes: true,
         childList: true,
-        characterData: true,
```

Comment:
any ideas how we could have a meaningful test for this?

Or we could add an example of code causing a failure before we made this change?

Reply:
User in Discord mentioned it was breaking possibly with a contenteditable attribute.

I wasn't going to write a test since it brings it in line with Alpine, and probably should have never been included to start with. https://github.com/alpinejs/alpine/blob/master/src/index.js#L58

But I'll see about writing a test now anyway. Shouldn't be too much work

Reply:
yeah fair enough, JSDOM has weird unimplemented bits and pieces so if it's not straightforward, we can drop an example into one of the HTML files and "test" that way.

Reply:
Should be good now. The test failed as expected then passed with the change